Update quickstart.md, added new 4th step making it 5 step documentation (#2641)
WordPress/openverse: documentation/frontend/guides/quickstart.md

   $ just node-install
   ```

4. If you encounter errors after this, you may need to compile just the Openverse ESLint plugin.

   ```console
   $ pnpm --filter='@openverse/eslint-plugin' build
   ```

   Or

   ```console
   $ pnpm -F '@openverse/eslint-plugin' build

>    ```

Reviewer: This seems an unnecessary repetition of the previous command. We could omit this.

   $ just node-install
   ```

4. Filtering allows you to restrict commands to specific subsets of packages.

>    You might want to try Filtering to compile just the Openverse ESLint plugin.

Reviewer 2: Slight rewording for clarity:
```suggestion
   If you encounter errors after this, you may need to compile just the Openverse ESLint plugin.
```

   ```console
   $ pnpm --filter='@openverse/eslint-plugin' build
   ```

   Or

   ```console

>    $ pnpm --F '@openverse/eslint-plugin' build

Reviewer 3: The short form of the option is with one dash only. Otherwise it looks good!
```suggestion
   $ pnpm -F '@openverse/eslint-plugin' build
```
